match2/commons/match.lua

```diff
@@ -100,7 +102,7 @@ function p._storeLegacy(parameters, rawOpponents, rawGames)
 	legacy.storeMatch(rawMatch)
 end
 
-function p._storePlayers(args, staticid, opponentIndex)
+function p._storePlayers(args, staticid, opponentIndex, storeInLPDB)
```

Comment:
Feels like perhaps this would be better as something set globally in the file, since these are private functions and we keep having to pass these on currently